Sources/UtiliKit/General/LanguageTool.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  LanguageTool.swift
+//  UtiliKit-iOS
+//
+//  Created by Russell Mirabelli on 10/11/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import Foundation
+
+@available(iOSApplicationExtension 11.0, *)
+class LanguageTool {
+
+    private let badverbs = ["be", "have", "do", "say", "get", "make", "go", "know", "take", "see", "come", "think", "look", "want", "give", "use", "find", "tell", "ask", "work", "seem", "feel", "try", "leave", "call"]
+
+    private let badnouns = ["way", "day", "thing", "place", "work", "case", "point", "company", "number"]
+
+    private let badadjectives = ["long", "little", "own", "other", "old", "right", "big", "high", "different", "small", "large", "next", "early", "young", "important", "few", "public", "bad", "same", "able"]
+
+    private let badprepositions = ["to", "of", "in", "for", "on", "with", "at", "by", "from", "up", "about", "into", "over", "after"]
+
+    private let badOthers = ["the", "a", "which", "be", "to", "of", "and", "a", "an", "this", "one", "can", "it", "if", "as", "but", "they", "let", "there", "should"]
+
+
+    func findSetOfWords(in str: String) -> [String] {
```

Comment:
Based on the logic below, this is returning the words in the provided String, less any of the "bad words" it might contain, correct? We might want to rename this to be a bit clearer.

```suggestion
    func goodWords(in str: String) -> [String] {
```